go.mod

```diff
@@ -1,26 +1,24 @@
-module github.com/cyriltovena/logger
+module cluster-logging-load-client/loadclient
 
-go 1.14
+go 1.16
 
 require (
 	github.com/cenkalti/backoff/v4 v4.1.0
 	github.com/cortexproject/cortex v1.2.1-0.20200803161316-7014ff11ed70
 	github.com/elastic/go-elasticsearch/v6 v6.8.10
+	github.com/go-kit/kit v0.10.0 // indirect
 	github.com/gogo/googleapis v1.2.0 // indirect
 	github.com/gogo/status v1.1.0 // indirect
-	github.com/google/go-cmp v0.5.2 // indirect
 	github.com/grafana/loki v1.6.1
 	github.com/mitchellh/go-homedir v1.1.0
 	github.com/prometheus/common v0.10.0
+	github.com/sirupsen/logrus v1.5.0 // indirect
 	github.com/spf13/cobra v1.1.1
 	github.com/spf13/viper v1.7.0
-	github.com/weaveworks/common v0.0.0-20200625145055-4b1847531bc9
-	golang.org/x/sys v0.0.0-20200828194041-157a740278f4 // indirect
-	golang.org/x/text v0.3.3 // indirect
-	golang.org/x/tools v0.0.0-20200910222312-571a207697e7 // indirect
+	golang.org/x/lint v0.0.0-20210508222113-6edffad5e616 // indirect
+	golang.org/x/tools v0.1.4 // indirect
```

Comment:
Lint tools don't belong here. To handle them out of your go module tree, you can sidestep and use tools like [Bingo](https://github.com/bwplotka/bingo/). Take a look on cluster-logging-operator or elasticsearch-operator.

Reply:
Added Bingo support 
DONE